site/content/en/references/kustomize/kustomization/replacements/_index.md

```diff
@@ -142,6 +142,29 @@ reject:
 - kind: StatefulSet
 ```
 
+Moreover, when the selected target is going to be transformed during the kustomization process,
```

Comment:
1) It felt kinda random to explain only about the behavior I fixed, so I added a bit more info about the functionality of `select` field in general so that it matches `reject`.
2) I used `namePrefix` for explaining `select` because I already used `nameSuffix` at `reject`.

e701ab511a4973fc8e3de85bc6ec0dcc4d4f1ab7